src/components/NewLexemeForm.vue

```diff
@@ -46,20 +46,26 @@ export default {
 		<lexical-category-input
 			v-model="lexicalCategory"
 		/>
-		<wikit-button
-			class="form-button-submit"
-			type="progressive"
-			variant="primary"
-		>
-			{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
-		</wikit-button>
+		<div>
+			<wikit-button
+				class="form-button-submit"
+				type="progressive"
+				variant="primary"
+			>
+				{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
+			</wikit-button>
+		</div>
 	</form>
 </template>
 
 <style scoped lang="scss">
 @import "@wmde/wikit-tokens/variables";
 
 .wbl-snl-form {
+	& > :not(:last-child) {
+		margin-block-end: $dimension-layout-xsmall;
+	}
```

Comment:
Personally, to recreate this grid functionality without getting into specificity loopholes of `:not`, we used to use the "owl eyes"

```suggestion
	& > * + * {
		margin-block-start: $dimension-layout-xsmall;
	}
```

Reply:
looks unusual, but why not. I'll update the PR

Reply:
FWIW, to me this looks much less readable than the old version :/